templates/crate/features.html

```diff
@@ -40,7 +40,7 @@
                             </li>
                         {%- else -%}
                             <li class="pure-menu-item">
-                                <span style="font-size: 13px;">Feature flags data are not available for this release.</span>
+                                <span style="font-size: 13px;">Feature flags data are not available for this release because it was built before features were collect by docs.rs.</span>.
```

Comment:
```suggestion
                                <span style="font-size: 13px;">Feature flags data are not available for this release because it was built before features were collected by docs.rs.</span>.
```

(The usage of plural data really threw me off, but I think it's actually correct).

Reply:
Happy to remove "data", I don't think it adds anything.